packages/manager/modules/sign-up/src/form/form.constants.js

```diff
@@ -30,8 +30,19 @@ export const MODEL_DEBOUNCE_DELAY = 500;
 
 export const OVH_SUBSIDIARY_ITEM_NAME = 'ovhSubsidiaryCreationForm';
 
+export const READY_ONLY_PARAMS = [
```

Comment:
may be `READY_RULES_PARAMS` ?

Reply:
In fact it is a mistake to type it. READY => READ